lib/osf-components/addon/components/contributors/user-search/widget/template.hbs

```diff
@@ -7,6 +7,7 @@
             @placeholder={{t 'registries.registration_metadata.add_contributors.search_placeholder'}}
             @value={{this.query}}
             @enter={{perform this.fetchUsers}}
+            {{on 'input' this.resetPage}}
```

Comment:
If this only resets on changes to input, does this mean that pressing the search button or hitting enter will generate the error if there's only one page? If there is more than one page, will that cause the next page to fetch?